<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelCellCurrent">
         <property name="text">
          <string>当前单元格</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCellText">
         <property name="text">
          <string>单元格内容</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSetRowAndColumn"/>
   <addaction name="actSetHeadAndTitle"/>
   <addaction name="actCellLocation"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actSetRowAndColumn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/230.bmp</normaloff>:/images/images/230.bmp</iconset>
   </property>
   <property name="text">
    <string>设置行数列数</string>
   </property>
  </action>
  <action name="actSetHeadAndTitle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/516.bmp</normaloff>:/images/images/516.bmp</iconset>
   </property>
   <property name="text">
    <string>设置表头标题</string>
   </property>
  </action>
  <action name="actCellLocation">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/304.bmp</normaloff>:/images/images/304.bmp</iconset>
   </property>
   <property name="text">
    <string>定位单元格</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
